apps/explorer/src/pages/address-result/AddressBalanceBreakdown.tsx

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    Collapsible,
+    formatDelegatedStake,
+    formatDelegatedTimelockedStake,
+    mapTimelockObjects,
+    TIMELOCK_IOTA_TYPE,
+    useBalance,
+    useFormatCoin,
+    useGetAllOwnedObjects,
+    useGetDelegatedStake,
+    useGetTimelockedStakedObjects,
+    useTotalDelegatedStake,
+} from '@iota/core';
+import { Divider, KeyValueInfo, Panel, Skeleton, Title, TitleSize } from '@iota/apps-ui-kit';
+import { useState } from 'react';
+
+const TOOLTIP_TEXT = 'This balance breakdown does not include unmigrated stardust funds.';
+interface BalanceBreakdownElement {
+    keyText: string;
+    value: string;
+    supportingLabel: string;
+    isLoading?: boolean;
+    isError?: boolean;
+    tooltipText?: string;
+}
+
+export function AddressBalanceBreakdown({ address }: { address: string }): React.JSX.Element {
+    const [open, setOpen] = useState(true);
+    const {
+        data: balance,
+        isLoading: isLoadingBalance,
+        isError: isBalanceErrored,
+    } = useBalance(address);
+
+    const [totalAvailableBalance, symbol] = useFormatCoin({
+        balance: balance?.totalBalance,
+    });
+
+    const {
+        data: delegatedStake,
+        isLoading: isLoadingDelegatedStakes,
+        isError: isDelegatedStakeErrored,
+    } = useGetDelegatedStake({
+        address,
+    });
+    const delegatedStakes = delegatedStake ? formatDelegatedStake(delegatedStake) : [];
+    const totalDelegatedStake = useTotalDelegatedStake(delegatedStakes);
+    const [formattedDelegatedStake] = useFormatCoin({
+        balance: totalDelegatedStake,
+    });
+
+    const {
+        data: timelockedStakedObjects,
+        isLoading: isLoadingTimelockedStakeObjects,
+        isError: isTimelockedStakedObjectsErrored,
+    } = useGetTimelockedStakedObjects(address);
+
+    const extendedDelegatedTimelockedStakes = formatDelegatedTimelockedStake(
+        timelockedStakedObjects || [],
+    );
+
+    const totalTimelockedStaked = extendedDelegatedTimelockedStakes?.reduce(
```

Comment:
this calculation is the same as for     `const totalDelegatedStake = useTotalDelegatedStake(delegatedStakes);`
maybe we can use the same hook here just update the params a bit(maybe rename it)
```
export function useTotalDelegatedStake(
    delegatedStake: ExtendedDelegatedStake[] | ExtendedDelegatedTimelockedStake[],
) {
    return useMemo(() => {
        if (!delegatedStake) return 0n;
        return delegatedStake.reduce((acc, curr) => acc + BigInt(curr.principal), 0n);
    }, [delegatedStake]);
}

```
and than
`    const totalTimelockedStaked = useTotalDelegatedStake(extendedDelegatedTimelockedStakes);
`